mail/mailutils/Portfile

```diff
@@ -82,15 +82,28 @@ variant tcp_wrappers description {Build with tcp_wrappers support} {
     depends_build-append    port:tcp_wrappers
 }
 
-variant python27 description {Enable support for Python 2.7 scripting} {
```

Comment:
there is no option anymore to enable Python scripting?

Reply:
not yet. mailutils is supposed to support python39, but i cant seem to make it work.
I'll keep trying to have python scripting working (but i guess i shall open a ticket).

Reply:
so what doesn't work? I suppose you tried
```
variant python description {Enable support for Python scripting} {
     configure.args-replace  --disable-python --enable-python
     configure.python        ${prefix}/bin/python2.7
     depends_lib-append      port:python39
}
```

Reply:
i am trying right with that stanza (I guess the python2.7 is a typo).

the configure phase recognize python39, but it fails to build with the following error during linking:
```
	ld: library not found for -lpython3.9
	clang: error: linker command failed with exit code 1 (use -v to see invocation)
	make[3]: *** [libmu_py.la <http://libmu_py.la/>] Error 1
```
ps i wrote about it in [this mail](https://lists.macports.org/pipermail/macports-dev/2021-May/043539.html)

Reply:
I've update the code above. The issue to me seems that it could be caused by the fact that it is looking for `python-config`. That file only exists if the user has done `port select python`; specifying the one installed by Python 3.9 is more correct for sure. I am trying to build it now so I don't know yet if this resolves the issue you are having, but it might be....

Reply:
i don't really know. 
i tried with all the different combination you can select via python_select.
i have also tried to add LDFLAGS and CPPFLAGS to be passed on in the build phase. without success. 
btw, those are the env vars i need to compile successfully from sources tarball out of the macports domain. 
let's see if you succeed, could be also a fail specific to my installation! 
cheers

Reply:
please try this, it does install the port for me and it seems to link correctly. But I don't know anything else about this port so don't know how to test it....
```
variant python description {} {
    set py_ver          3.9
    set py_ver_nodot    [string map {. {}} ${py_ver}]

    configure.ldflags-append \
                        -L${frameworks_dir}/Python.framework/Versions/${py_ver}/lib
    configure.args-append \
                        PYTHON_CONFIG=${prefix}/bin/python${py_ver}-config
    configure.args-replace  \
                        --disable-python --enable-python
    configure.python    ${prefix}/bin/python${py_ver}
    depends_lib-append  port:python${py_ver_nodot}
}
```

Reply:
it does work. i am going to amend the PR and commitment message. thanks.